### PR TITLE
Angular ivy

### PR DESCRIPTION
**Pull request by mhartington**

### CODE REVIEW

File: angular/src/directives/navigation/ion-back-button.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export class IonBackButtonDelegate {
    */
   @HostListener('click', ['$event'])
   onClick(ev: Event) {
+    console.log('clicked', ev);
```
Comment (dylanvdmerwe) [inline angular/src/directives/navigation/ion-back-button.ts]: Probably shouldn't be here?

**Comment by mhartington** [comment]
Needs https://github.com/ionic-team/stencil/pull/2033 to be merged/resolved

**Comment by brandyscarney** [comment]
https://github.com/ionic-team/stencil/pull/2033 has been merged, need a new pre-release for testing

File: .scripts/common.js [hunk 2]
```diff
@@ -137,7 +138,7 @@ function preparePackage(tasks, package, version, install) {
   }
 
   if (package !== 'docs') {
-    if (package !== 'core') {
+    if (package !== 'core' || package !== 'packages/angular-server') {
```
[the remark this answers is not in the document]
Comment (mhartington) [reply .scripts/common.js]: No, this is correct as it is.

[the remark this answers is not in the document]
Comment (vially) [reply .scripts/common.js]: Most probably I'm missing something, but is there any reason to keep the `if` statement in there if the condition is always `true`?

**Comment by brandyscarney** [comment]
I updated the release script to fix some of my comments. The only outstanding comment is related to the order of updating the core dependencies and running `npm link`. This may cause issues, I want to say that's why they were separate.

Overall testing of the dev build seems to be going well, so that's my only concern!

**Comment by brandyscarney** [comment]
:shipit:

**Pull request merged**